SETUP.md

```diff
@@ -27,7 +28,33 @@ These steps should be performed in a command-line shell (terminal), starting in
 Setup is supported on Linux, macOS, and Windows.
 Windows setup requires [Git Bash](https://gitforwindows.org/) or [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/faq).
 
-## Configuration
+## Using setup script
+Creating a new manuscript using GitHub actions, the recommended default CI service (see below), can be achieved easily using the [setup script](https://github.com/manubot/rootstock/setup.bash). This simply runs the steps detailed below in the Manual Configuration.
+
+First, you must manually create an empty GitHub repository at <https://github.com/new>.
+Do not initialize the repository, other than optionally adding a Description.
+
+Next, copy `setup.bash` to the directory where you would like to clone your manuscript repo, then make it executable.
+
+Then run the script using the `OWNER` and `REPO` details for your manuscript repo location:
+i.e. `https://github.com/OWNER/REPO`.
+
+````sh
+# create file in dir to contain the manuscript repo
+touch setup.bash
+
+# Use your favourite editor to copy and paste in the
+# contents of https://github.com/manubot/rootstock/setup.bash
+
+# Make script executable
+chmod +x setup.bash
```

Comment:
you probably considered:

```bash
curl URL_TO_setup.bash | bash
```

This would be more convenient, but perhaps less portable and doesn't give the user the chance to audit the code.

Reply:
TBH I didn't think about this as I'm relatively new to linux!
This is much neater so I will change the instructions to use curl, syntax has to be a bit different to allow user interaction once the script starts.

Reply:
> I didn't think about this as I'm relatively new to linux!

A small note here, this command should not be limited to working on just linux. Ideally it will work on macOS, linux, and Windows Bash shells. The actual setup script can require bash, but users will likely enter the initial command from some non-bash shells.

> TBH I didn't think about this as I'm relatively new to linux!

Welcome! I've been using linux for decades, but still don't know many of the shell script constructs and syntaxes you've masterfully employed. haha

Reply:
Haha thanks. 

> I want to respectful of your time, so let us know whether you are running out of contribution time.

I'm happy to continue working on this but it maybe a bit slow depending on how much I've got on that's all.